apps/api-journeys/src/modules/trigger/trigger.spec.ts

```diff
@@ -0,0 +1,111 @@
+import { testkit, gql } from 'graphql-modules'
+import { schemaBuilder } from '@core/shared/util-graphql'
+import {
+  videoTriggerModule,
+  journeyModule,
+  blockModule,
+  actionModule
+} from '..'
+import dbMock from '../../../tests/dbMock'
+import { v4 as uuidv4 } from 'uuid'
+import { Block, ThemeName, ThemeMode } from '.prisma/api-journeys-client'
+import { DocumentNode, ExecutionResult } from 'graphql'
+
+describe('TriggerModule', () => {
```

Comment:
```suggestion
describe('VideoTriggerModule', () => {
```

Can you make sure to update all instances of trigger to videoTrigger?